docs/en/api/css/data-type.md

```diff
@@ -1,8 +1,7 @@
 ---
 overview: true
 overviewHeaders: [2]
+title: Data Type
```

Comment:
why do we need to change this?

Reply:
@SoonIter The H1 was supposed to be hidden for pages with `overview: true`. This bug was probably a breaking change introduced in Rspress v2